test/d_do_test.d

```diff
@@ -565,6 +568,16 @@ int tryMain(string[] args)
     if (!gatherTestParameters(testArgs, input_dir, input_file, envData))
         return 0;
 
+    // Clear the DFLAGS environment variable if it was specified in the test file
+    if (testArgs.dflags !is null)
+    {
+        if (testArgs.dflags != "")
+            throw new Exception("The DFLAGS test argument must be empty: It is '" ~ testArgs.dflags ~ "'");
+
+        environment["DFLAGS"] = "";
+        writefln("DFLAGS='%s'", environment["DFLAGS"]);
```

Comment:
This print statement is going to go to the main test log, which may not be what was intended.

Reply:
Ah OK.  I find it helpful to include a comment in these cases so I don't forget.
```
// REMOVE ME BEFORE MERGE
writefln("...");
```

It's funny because I mainly develop on windows, but after hours of failure I still haven't been able to get the DMD test suite running on windows.  So I have to SSH into my linux box to run the test suite.  I don't think there is even a WIKI or documentation on how to get the test suite running on windows.  Definitely something we should setup...then maybe I could finally get it to work :)

Reply:
https://issues.dlang.org/show_bug.cgi?id=18413